tests/installation/grub_test.pm

```diff
@@ -80,9 +73,26 @@ sub run() {
         }
         send_key 'ret';
     }
-    if (get_var("XEN")) {
+    elsif (get_var("XEN")) {
         send_key_until_needlematch("bootmenu-xen-kernel", 'down', 10, 5);
     }
+    elsif (check_var('ARCH', 'aarch64') && check_var('DISTRI', 'sle') && get_var('PLYMOUTH_DEBUG')) {
+        send_key 'e';
+        # Move to end of kernel boot parameters line
+        send_key_until_needlematch "linux-line", "down";
+        send_key "end";
+        if (check_screen("linux-line-matched")) {
+            # remove "splash=silent quiet showopts"
+            for (1 .. 28) { send_key "backspace" }
```

Comment:
That's why I check_screen - to make sure I can use those magic numbers (better suggestion?)